client/src/ui/atoms/TextField.tsx

```diff
@@ -6,7 +6,6 @@ interface Props extends InputHTMLAttributes<HTMLInputElement> {
     id: string;
     invalid?: boolean;
     labelText: string;
-    placeholder?: string;
```

Comment:
Why are we removing placeholder attribute?

Reply:
its propogated through the `{...rest}` spread so doesn't need to be forwarded like this.